### PR TITLE
Node js

### PR DESCRIPTION
**Pull request by abp437**
# Basic NodeJS Server Creation and Routing

### CODE REVIEW

File: Node JS/router.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+			response.writeHead(404, 'Error Encountered', {'Content-Type': 'text/html'});
+		} else {
+			response.write(data);
+		}
+		response.end();
+	});
+}
+
+module.exports = {
+	handleRequest: function(request, response) {
```
Comment (Shrikant9) [inline Node JS/router.js]: If you're only handling one route, you don't need a separate file.

File: Node JS/server.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+var http = require('http');
+var router = require('./router');
+
+http.createServer(router.handleRequest).listen(8000);
```
Comment (Shrikant9) [inline Node JS/server.js]: Do this & the routing with Express instead.

File: Node JS/index.html [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```
Comment (Shrikant9) [inline Node JS/index.html]: Try to use EJS/jade(pug) templating instead.

**Comment by abp437** [comment]
Closing this PR and opening a new one

**Pull request closed without merging**